package.json

```diff
@@ -294,6 +309,18 @@
                 {
                     "command": "aks.refreshSubscription",
                     "when": "never"
+                },
+                {
```

Comment:
can we nicely say somewhere that if user doesn't have repo or work folder open, draft commands wouldn't show up or more in positive way please open up folder for draft commands ? I know our UX person struggled with this a bit in the beginning, just a thought. If it's overkill we can ignore.